docker-compose-devel.yml

```diff
@@ -1,28 +1,22 @@
+---
 version: '2.2'
 services:
   development:
-    image: ${IMAGE:-quay.io/3scale/s2i-openresty-centos7:1.19.3.6-20-centos8}
+    image: ${IMAGE:-quay.io/3scale/apicast-ci:openresty-1.19.3}
     depends_on:
       - redis
-    working_dir: /home/centos/
-    volumes:
-      - .docker/lua_modules:/home/centos/lua_modules
-      - .docker/local:/home/centos/local
-      - .docker/vendor/cache:/home/centos/vendor/cache
-      # no need to access those from docker
-      - /home/centos/.docker
-      - /home/centos/.git
+    working_dir: /opt/app-root/src/
```

Comment:
This breaks existing envrionments because the permissions are not the same, either need to take care of this ourselves OR document that the project should just be recreated.

Reply:
I  think I get your point. 

If you run "make development", the image being loaded will be `quay.io/3scale/apicast-ci:openresty-1.19.3` and there should not be any permission issue. 

Only  if you run "make development IMAGE=mycustom_image", then you need to make sure that "my_custom_image" is ubi8 based and has been built with "make dev-build" target. Usually the devel image is not custom image. Only when we neeed to upgrade some dep existing in the devel image, we need to test the new devel image with a custom image before pushing a new release